tests/driver_bme280/Makefile

```diff
@@ -0,0 +1,15 @@
+APPLICATION = driver_bme280
+include ../Makefile.tests_common
+
+USEMODULE += bme280
+USEMODULE += xtimer
+
+# set default device parameters in case they are undefined
+TEST_I2C                  ?= I2C_0
+TEST_I2C_ADDR             ?= 0x77
+
+# export parameters
+CFLAGS += -DTEST_I2C=$(TEST_I2C)
+CFLAGS += -DTEST_I2C_ADDR=$(TEST_I2C_ADDR)
```

Comment:
Command line as in: "via environment variable CFLAGS" ?
For example this?

    CFLAGS=-DBME280_PARAM_I2C_ADDR=0x76 make -C tests/driver_bme280 BOARD=sodaq-autonomo